packages/plugin-ext/src/hosted/browser/worker/worker-main.ts

```diff
@@ -102,10 +116,10 @@ const pluginManager = new PluginManagerExtImpl({
             } else {
                 foreign.push({
                     pluginPath: pluginModel.entryPoint.backend!,
-                    pluginFolder: plg.source.packagePath,
+                    pluginFolder: pluginModel.packagePath,
                     model: pluginModel,
                     lifecycle: pluginLifecycle,
-                    rawModel: plg.source
+                    rawModel: stubPluginPackage(pluginModel)
```

Comment:
do we still need this rawModel ?

Reply:
It will be bigger refactoring to make it optional. There are different clients in the plugin host, assuming that it is always present, i.e. debug ext and vscode backend init script.

Reply:
it can be done in a follow-up but it's like there is a big hack by faking the raw stuff and let think it is still there.

Reply:
I can also do `rawModel: undefined!`, so if frontend plugins try to access it they will get `undefined`. Would it work?